content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -25,16 +25,16 @@ This document describes the current state of `PersistentVolumes` in Kubernetes.
 
 ## Introduction
 
-Managing storage is a distinct problem from managing compute. The `PersistentVolume` subsystem provides an API for users and administrators that abstracts details of how storage is provided from how it is consumed. To do this we introduce two new API resources:  `PersistentVolume` and `PersistentVolumeClaim`.
+Managing storage is a distinct problem from managing compute. The `PersistentVolume` subsystem provides an API for users and administrators that abstracts details of how storage is provided and how it is consumed. To do this we introduce two new API resources:  `PersistentVolume` and `PersistentVolumeClaim`.
```

Comment:
Please remove "for users and administrators"

"is provided and consumed", please remove the extra words.

suggest "To configure storage, you create a PersistentVolume and PersistentVolumeClaim."

Reply:
I experimented with various versions without any success. So I would like leave this refactoring to  more experienced people.